.azure-pipelines/ci-build.yml

```diff
@@ -616,11 +616,23 @@ extends:
                 displayName: "Get Kiota's version-number from .csproj"
               - pwsh: $(Build.SourcesDirectory)/scripts/update-vscode-releases.ps1 -version $(artifactVersion)$(versionSuffix) -packageJsonFilePath $(Build.SourcesDirectory)/vscode/microsoft-kiota/package.json -runtimeFilePath $(Build.SourcesDirectory)/vscode/npm-package/runtime.json -binaryFolderPath $(Build.ArtifactStagingDirectory)/Binaries
                 displayName: "Update VSCode extension version-number"
-              - script: npm i -g @vscode/vsce
-                displayName: "Install vsce"
-              - script: npm ci
+              - script: |
+                  npm i -g @vscode/vsce
+                  npm i -g rimraf
+                displayName: "Install Global dependencies"
+              - script: |
+                  cd npm-package
+                  npm install
+                  npm run package
+
+                  cd ..
```

Comment:
why not run separate script blocks instead for each of the installs? So we have a clear idea of which one failed when it happens?

Reply:
Done